pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.black]
-target_version = ['py311']
```

Comment:
Have you forgotten to refresh/commit the lock file? I would expect a change like this to cause a bunch of updates to the lock file.

Reply:
Just realized my comment wasn't clear enough: I wasn't referring to that specific line. Let me rephrase it:

Have you refreshed the lock file from a system with python 3.12?

Reply:
I tried to install it in python3.12 venv  both locally and in integration tests and it was running fine with the current lock file. But I can regenerate the new one if you want.